### PR TITLE
Update old wiki links

### PR DESCRIPTION
**Pull request by Piinks**
Part of https://github.com/flutter/flutter/issues/145009

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -104,35 +104,35 @@ that describe these methodologies. The following list is ordered: you
...
    practice to do before coding more intricate changes.
-   See also our [guidance for writing design docs](https://github.com/flutter/flutter/wiki/Design-Documents).
+   See also our [guidance for writing design docs](https://github.com/flutter/flutter/blob/master/docs/contributing/Design-Documents.md).
 
 [![How to contribute to Flutter](https://img.youtube.com/vi/4yBgOBAOx_A/0.jpg)](https://www.youtube.com/watch?v=4yBgOBAOx_A)
 
 In addition to the documents, there is a video linked above on **How to contribute to Flutter**
 from the [Flutter](https://youtube.com/c/flutterdev) YouTube channel,
-there are many pages on [our Wiki](https://github.com/flutter/flutter/wiki/),
+there are many pages on [our Wiki](https://github.com/flutter/flutter/tree/master/docs),
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:135]: This should probably not say "our Wiki" any more.

**Review by stuartmorgan — APPROVED** [review]
LGTM with nit.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -41,27 +41,27 @@ useful for our engineers.
...
-1. Join the #hackers-triage [Discord channel](https://github.com/flutter/flutter/wiki/Chat).
+1. Join the #hackers-triage [Discord channel](https://github.com/flutter/flutter/blob/master/docs/contributing/Chat.md).
 
 2. Read [our code of conduct](CODE_OF_CONDUCT.md), which stipulates explicitly
    that everyone must be gracious, respectful, and professional. If you're helping out
    with triage, you are representing the Flutter team, and so you want to make sure to
    make a good impression!
 
-3. Help out as described in our wiki: https://github.com/flutter/flutter/wiki/Triage
+3. Help out as described in our wiki: https://github.com/flutter/flutter/blob/master/docs/triage/README.md
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:51]: I missed a "wiki" reference here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -41,27 +41,27 @@ useful for our engineers.
...
    You won't be able to add labels at first, so instead start by trying to
    do the other steps, e.g. trying to reproduce the problem and asking for people to
    provide enough details that you can reproduce the problem, pointing out duplicates,
    and so on. Chat on the #hackers-triage channel to let us know what you're up to!
 
 4. Familiarize yourself with our
-   [issue hygiene](https://github.com/flutter/flutter/wiki/Issue-hygiene) wiki page,
+   [issue hygiene](https://github.com/flutter/flutter/blob/master/docs/contributing/issue_hygiene/README.md) wiki page,
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:58]: And here.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -41,27 +41,27 @@ useful for our engineers.
...
    which covers the meanings of some important GitHub labels and
    milestones.
 
 5. Once you've been doing this for a while, someone will invite you to the flutter-hackers
    team on GitHub and you'll be able to add labels too. See the
-   [contributor access](https://github.com/flutter/flutter/wiki/Contributor-access) wiki
+   [contributor access](https://github.com/flutter/flutter/blob/master/docs/contributing/Contributor-access.md) wiki
```
Comment (stuartmorgan) [inline CONTRIBUTING.md:64]: And here.

Looks like we could just replace "wiki" and "wiki page" with "docs" in all of these.

**Review by stuartmorgan — APPROVED** [review]
LGTM

I missed some other reference to the wiki; up to you if you want to fix them now or not.

**Comment by Piinks** [comment]
I'll just fix them now, I'll need another stamp though since it is dismissed whenever I push a new commit in this repo. :)

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Pull request merged**